x-pack/plugins/cases/public/components/user_actions_activity_bar/index.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+
+import { FilterActivity } from './filter_activity';
+import { SortActivity } from './sort_activity';
+import type { FilterType, SortOrderType } from './types';
+import type { CaseUserActionsStats } from '../../containers/types';
+
+export interface Params {
```

Comment:
nit: In other components, it may be difficult to understand what the type is about if we import a generic name like `Param`. We can either rename it to `UserActivityParams` or do `import { Params as UserActivityParams }` to the components. Same for `FilterType` and `SortOrderType`.